Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM openjdk:8u92-alpine
+
+ARG JAR_FILE
+ARG CONF_FILE
+ARG PROP_FILE
+ARG MYSQL_CONNECTOR_VERSION=5.0.8
+
+RUN apk add --update bash curl unzip && \
+    rm -rf /var/cache/apk/*
+
+# Install MySQL Connector
+WORKDIR /opt/holdmail/lib
```

Comment:
I copied the existing Dockerfile into the root.  I'm undecided if we still need the existing docker setup, its good because it includes a docker-compose file, but that may not be long for this earth in favor of k8s.